recipes/csvlint/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "csvlint" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 886c87441af105065e620386434ae2cd9d9c69cfcd532335ae52f1cd15f419c8
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: true  # [win]
+
+requirements:
+  host:
+    - ruby >=2.5,<2.6
+  run:
+    - ruby >=2.5,<2.6
+    - rb-rainbow
+    - rb-open_uri_redirections
+    - rb-activesupport
+    - rb-addressable
+    - rb-typhoeus
+    - rb-escape_utils
+    - rb-uri_template
+    - rb-thor
+    - rb-rack
+    - rb-net-http-persistent
+    - rb-concurrent-ruby <1.1
```

Comment:
The pin works for now, but it probably makes sense to pin the ruby version in the [`rb-concurrent-ruby` feedstock](https://github.com/conda-forge/rb-concurrent-ruby-feedstock/blob/main/recipe/meta.yaml).